README.md

```diff
@@ -221,7 +221,7 @@ versions in the 3 series may not be supported by other p4lang projects, such as
 well, but all our CI testing is done with version 3.2.0. After cloning protobuf
 and before you build, check-out version 3.2.0:
 
-`git checkout v3.2.0`
+`git checkout v3.6.1`
```

Comment:
There are multiple occurrences of the version number 3.2.0 in the README.  It would be very confusing only to update this one.

Reply:
Since p4c is now using protobuf 3.6.1, would folks mind if I changed other instances of 3.2.0  to 3.6.1?  thanks.hh

Reply:
I just verified p4lang/PI and even this repo has upgraded to using protobug 3.6.1.  See

https://github.com/p4lang/PI/commit/1539ecd8a50c159b011d9c5a9c0eba99f122a845

Reply:
As far as I know, everything still works with protobuf 3.2.0, too.  @antoninbas Any opinion on what the p4c README should say about protobuf versions?  In particular, OK to replace all mentions of 3.2.0 with 3.6.1?  It doesn't seem critical to make such a change if both work, but if other open source P4 tools are "officially supporting" version 3.6.1, it seems to make sense to update all relevant README/install scripts eventually.

Reply:
One may note that p4lang/PI documentation has already moved to using protobuf 3.6.1.